pagecarton/core/library/Ayoola/Application.php

```diff
@@ -1196,20 +1195,16 @@ public static function display()
 					//	change requested url
 					$requestedUri = self::getRequestedUri();
 				//	var_export( $requestedUri );
-				//	var_export( $multiSiteDir );
 					$requestedUri = explode( $multiSiteDir, $requestedUri );
-				//	var_export( $requestedUri ); 
+			//		var_export( $requestedUri );
 					array_shift( $requestedUri );
-				//	var_export( $requestedUri );  
-				//	$requestedUri = implode( $multiSiteDir, $requestedUri );
-					$requestedUri = array_shift( $requestedUri );
-				//	var_export( $requestedUri );  
+					$requestedUri = implode( '', $requestedUri );

```

Comment:
I am not sure this line will work as intended. A lot of conditions necessitated the use of array_shift instead of implode. This might break some applications. Let's leave it out unless it was used to fix a bug.